kore/src/Kore/Variables/AsVariable.hs

```diff
@@ -0,0 +1,9 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+
+-}
+module Kore.Variables.AsVariable where
+
+class AsVariable f where
+    asVariable :: f variable -> variable
```

Comment:
I've removed it and replaced it with a `foldMapVariable :: (variable -> a) -> UnifiedVariable variable -> a` which is meant to be used to extract variable-related info from a `UnifiedVariable`, although it can be abused to obtain `asVariable = foldMapVariable id`.